docs/src/main/sphinx/release/release-453.md

```diff
@@ -138,3 +138,7 @@
 ## SQL Server connector
 
 * Add support for the `execute` procedure. ({issue}`22556`)
+
+## SPI
+
+* {{breaking}} Add `SystemAccessControl.getTableColumnMasks` as replacement for `SystemAccessControl.getColumnMask`. ({issue}`21997`)
```

Comment:
This is not (yet) a breaking change. The old method is still available.

Reply:
Oh .. so it was only soft deprecated?

Reply:
@mosiac1 ?

Reply:
confirmed on slack